src/solver/optim-model-filler/include/antares/solver/optim-model-filler/ComponentFiller.h

```diff
@@ -52,33 +53,36 @@ class ComponentFiller: public Optimisation::LinearProblemApi::LinearProblemFille
     ComponentFiller() = delete;
     ComponentFiller(ComponentFiller& other) = delete;
     /// Create a ComponentFiller for a Component
-    explicit ComponentFiller(const Study::SystemModel::Component& component);
+    explicit ComponentFiller(const Study::SystemModel::Component& component,
+                             const std::vector<Study::SystemModel::Connection>& connections);
 
     void addVariables(Optimisation::LinearProblemApi::ILinearProblem& pb,
                       Optimisation::LinearProblemApi::ILinearProblemData& data,
                       Optimisation::LinearProblemApi::FillContext& ctx) override;
 
-    void addStaticConstraint(Optimisation::LinearProblemApi::ILinearProblem& pb,
-                             const LinearConstraint& linear_constraint,
-                             const std::string& constraint_id) const;
-
-    void addTimeDependentConstraints(Optimisation::LinearProblemApi::ILinearProblem& pb,
-                                     const std::vector<LinearConstraint>& linear_constraints,
-                                     const std::string& constraint_id) const;
-
     void addConstraints(Optimisation::LinearProblemApi::ILinearProblem& pb,
                         Optimisation::LinearProblemApi::ILinearProblemData& data,
                         Optimisation::LinearProblemApi::FillContext& ctx) override;
+
     void addObjective(Optimisation::LinearProblemApi::ILinearProblem& pb,
                       Optimisation::LinearProblemApi::ILinearProblemData& data,
                       Optimisation::LinearProblemApi::FillContext& ctx) override;
 
     VariableDictionary variableDictionary;
 
 private:
+    void addStaticConstraint(Optimisation::LinearProblemApi::ILinearProblem& pb,
+                             const LinearConstraint& linear_constraint,
+                             const std::string& constraint_id) const;
+
+    void addTimeDependentConstraints(Optimisation::LinearProblemApi::ILinearProblem& pb,
+                                     const std::vector<LinearConstraint>& linear_constraints,
+                                     const std::string& constraint_id) const;
```

Comment:
These 2 functions used to be **public**. 
They were turned to **private**, because used only internally to the class.